powersimdata/scenario/check.py

```diff
@@ -0,0 +1,14 @@
+from powersimdata.scenario.analyze import Analyze
+from powersimdata.scenario.scenario import Scenario
+
+
+def _check_scenario_is_in_analyze_state(scenario):
+    """Ensure that scenario is a Scenario object in the analyze state.
```

Comment:
I think we want an empty line after the description.

Reply:
Done.